pkg/v14/configmap/desired.go

```diff
@@ -178,10 +178,6 @@ func exporterValues(configMapValues ConfigMapValues) ([]byte, error) {
 }
 
 func ingressControllerValues(configMapValues ConfigMapValues, hasLegacyIngressController bool) ([]byte, error) {
-	// controllerServiceEnabled needs to be set separately for the chart
-	// migration logic but is the reverse of migration enabled.
-	controllerServiceEnabled := !configMapValues.IngressController.MigrationEnabled
```

Comment:
This is no longer true so a separate flag is passed to the configmap service in pkg from the configmap resources.